### PR TITLE
Add password-rules for snnow.ca

### PR DESCRIPTION
**Pull request by roustem**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

#### for password-rules.json
- [x] The given rule isn't particularly standard and obvious for password managers
- [x] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
- [x] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)

20210929_140717@2x

### CODE REVIEW

File: quirks/password-rules.json [hunk 1]
```diff
@@ -566,6 +566,9 @@
     "secure.orclinic.com": {
         "password-rules": "minlength: 6; maxlength: 15; required: lower; required: digit; allowed: ascii-printable;"
     },
+    "secure.snnow.ca": {
+        "password-rules": "minlength: 7; maxlength: 16; required: digit; allowed: lower, upper, digit;"
```
Comment (rmondello) [inline quirks/password-rules.json:570]: You can omit “digit” from the allowed. All required classes are allowed.

**Review by rmondello — CHANGES_REQUESTED** [review]
Just a small nit, and then we’ll get this merged. :)

File: quirks/password-rules.json (hunk 1 above)
Comment (roustem) [reply quirks/password-rules.json]: Thanks!

**Review by rmondello — APPROVED** [review]

**Comment by rmondello** [comment]
Thanks @roustem! :D

**Pull request merged**